MyString.java

```diff
@@ -18,13 +18,43 @@ public static void main(String[] args) {
 
     /** Returns the lowercase version of the given string. */
     public static String lowerCase(String str) {
-        // Replace the following statement with your code
-        return null;
+        String news = "";
+        for(int i =0; i<str.length(); i++){
+            if (str.charAt(i)>=65&&str.charAt(i)<=90) {
+                 news+=(char)(str.charAt(i)+32);
+            }
```

Comment:
1. use chars rather than ASCII values
2. +32 is unclear, use '(char) (str.charAt(i) - 'A' + 'a')